src/vsr/replica_format.zig

```diff
@@ -154,7 +154,10 @@ test "format" {
 
     var storage = try Storage.init(
         allocator,
-        superblock_zone_size + constants.journal_size_headers + constants.journal_size_prepares,
+        superblock_zone_size +
+            constants.journal_size_headers +
+            constants.journal_size_prepares +
```

Comment:
```suggestion
            constants.journal_size_max
```

Also, it should probably be just `constants.journal_size`? The size is not variable